package.json

```diff
@@ -15,7 +15,8 @@
     "ramda": "^0.27.1",
     "tslib": "^2.3.1",
     "yaml": "^1.10.2",
-    "yeoman-generator": "^5.5.2"
+    "yeoman-generator": "^5.5.2",
+    "yo": "^4.3.0"
```

Comment:
This shouldn't be in the dependencies of the project.

Reply:
indeed x(